static/js/src/advantage/subscribe/react/components/Form/ProductType/ProductType.tsx

```diff
@@ -48,9 +48,9 @@ const PublicCloudInfo = {
     name: "Azure",
     CTA: [
       {
-        CTAName: "Ubuntu Pro in Azure marketplace",
+        CTAName: "in-place upgrade to Ubuntu Pro",
```

Comment:
"in-place" should be capitalised to "In-place"

Reply:
Updated but Lech said don't change the i to capital as microsoft has it with lower case but I found microsoft uses capital I... so I'm checking with Lech again.